src/components/Patient/PatientRegister.tsx

```diff
@@ -656,7 +656,7 @@ export const PatientRegister = (props: PatientRegisterProps) => {
             ? formData.last_vaccinated_date
             : null
           : null,
-      name: startCase(toLower(formData.name)),
+      name: formData.name ? startCase(formData.name.toLowerCase()) : "",
```

Comment:
we've discussed this to not transform the patient's name right? Let's keep whatever the user has inputted as is, and instead enable `autoCapitalize` prop in the respective input element to hint virtual/touch keyboards to suggest capitalizing using auto correct.

Reply:
ok ok